theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/LengthIndicator/LengthIndicator.tsx

```diff
@@ -239,10 +240,12 @@ function useDragBulge(node: HTMLDivElement | null, props: IProps) {
           tempTransaction.discard()
           tempTransaction = undefined
         }
+
         tempTransaction = getStudio()!.tempTransaction(({stateEditors}) => {
+          const newLength = initialLength + delta
           stateEditors.coreByProject.historic.sheetsById.sequence.setLength({
             ...sheet.address,
-            length: initialLength + delta,
+            length: isGreaterThanZero(newLength) ? newLength : 0,
```

Comment:
Fixing bug that allowed this to be dragged off the screen. Was there an issue for this?